### ISSUE TITLE
[1.9.24] iPhone 6 Plus / 10.0.1, iPad Mini 3 / 9.3.4 - Error "404" is displayed when user cancel the "Instagram" authorization.

### ISSUE THREAD

**Issue by testershub** [Lomotif/lomotif-ios-issues#764]
Frequency: Every time

Action:
1: Launch the app.
2: Tap on "START" and tap on "Instagram".
3: Login with new credentials **(Which is not used for lomotif yet)**
4: Authorization pop up will be displayed.
5: Tap on "Cancel" button on pop up.
6: Check if error 404 is displayed.

Expected result: User should be navigated to "Album" screen when user cancel the "Instagram" authorization.

Actual result: Error 404 is displayed when user cancel the "Instagram" authorization.

Refer screenshot for more details.

**Comment by testershub** [comment]
![404-error](https://cloud.githubusercontent.com/assets/18608449/18829780/7e387940-83fb-11e6-9413-c58b7b0f8590.jpg)

**Comment by hiteshbheda** [comment]
@xiuie I can't reproduce this issue for build 1.9.24 on iPhone 6 / iOS 10.0.2

When I enter credentials and click Login button, it directly navigates me to Instagram assets.

**Comment by testershub** [comment]
@hiteshbheda Have you used account which is not authorized for lomotif app yet?

**Comment by hiteshbheda** [comment]
@testershub Yes, I have tried with instagram account which is not yet used with Lomotif. I'll again try to verify this issue by creating new Instagram account.

**Comment by Dipamit** [comment]
@xiuie : I am able to reproduce this issue on iPhone 5/9.3.4 and iPhone 5/10.0.1 for build 1.9.25

Note: I used Instagram account which is authorized with Lomotif, Getting Error 404

**Comment by vysohanych** [comment]
@xiuie I can reproduce this issue on build 1.9.25 on iPhone 5S / iOS 9.3.5

**Comment by testershub** [comment]
@xiuie I can still reproduce this issue for build [1.9.27] on iPhone 5S / 9.3.5.
